docs/admin/auth/methods.rst

```diff
@@ -156,6 +156,11 @@ It's recommended to have the ``alg`` (`Algorithm parameter`_)  in the header.
 If it's provided both in the token and in the response from the JWK endpoint,
 both values are compared and in case of a mismatch the token is rejected.
 
+JWT authentication requires retrieving public keys from a `JWK endpoint`_.
+These public keys can be cached by CrateDB by setting ``cache-control: max-age``
```

Comment:
```suggestion
These public keys can be cached by CrateDB by setting ``Cache-Control: max-age``
```

While http headers are case insensitive, thats how it is referred to in the specificaiton

Reply:
👍